roles/confluent.control_center/defaults/main.yml

```
@@ -48,4 +48,4 @@ control_center_service_unit_overrides:
 ### Time in seconds to wait before starting Control Center Health Checks.
 control_center_health_check_delay: 30
 
-control_center_secrets_protection_file: /var/ssl/private/control-center-security.properties
+control_center_secrets_protection_file: "{{ ssl_file_directory }}control-center-security.properties"
```

```suggestion
control_center_secrets_protection_file: ""{{ (ssl_file_directory, 'control-center-security.properties') | path_join }}"
```

To avoid the requirement ^^ (MUST end in a backslash)

---

yes that is way better! 
But it says "New in version 2.10." ... i think we still have some users on 2.7, even though they are not supposed to

I don't think we could ask anyone to update ansible for a 6.1.x change, but maybe with 6.2.0

I suppose well be asking ppl to change to 2.11 soon though...

---

another option... i've been doing this elsewhere is to strip the last slash. I could strip it w regex and then add it back.
```
_ssl_file_directory: "{{ ssl_file_directory |regex_replace('\\/$', '') }}/"
```

---

I went w the second approach, just so this change would not require users to do anything (upgrade ansible). Good suggestion on removing that requirement. LMK what you think